retis/src/core/tracking/bpf/include/skb_tracking.h

```diff
@@ -49,7 +49,7 @@ struct tracking_info {
 	u64 last_seen;
 	/* Original head address; useful when the head is invalidated */
 	u64 orig_head;
-};
+} __binding;
```

Comment:
The comment above the definition "Please keep in sync with its Rust counterpart..." can be removed. There are many other examples.